lib/buffer.js

```diff
@@ -157,6 +157,12 @@ function palloc(that, length) {
   var buf = sliceOnto(allocPool, that, start, end);
   poolOffset = end;
 
+  // Ensure aligned slices
+  if (poolOffset & 0x7) {
+    poolOffset |= 0x7;
```

Comment:
@mscdex in other words, I'm just setting all bits that are lower than alignment value and increment the number to align it.